draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -994,16 +1100,21 @@
 
 	  <t> While the mechanism detailed in this document aims to provide dynamic metrics like Capacity Availability Index, Site Delay Prediction Index, Service Delay Prediction Index, and Raw Measurement to optimize path selection, it is essential to consider the broader implications of metric-induced churn. Particularly, in the context of routes used for BGP nexthop resolution (e.g., labeled unicast), frequent changes in these metrics can lead to significant churn not only for the prefixes carrying the data but also for dependent routes.</t>
 
-      <t>This behavior is analogous to the impacts observed with RSVP auto-bandwidth, which can introduce considerable instability within a network. Such route churn can propagate through the network, causing a cascade of updates and potential route flaps, thereby affecting overall network stability and performance.</t>
+      <t>This behavior is analogous to the impacts observed with RSVP auto-bandwidth, which can introduce considerable instability within a network. Such route churn can propagate through the network, causing a cascade of UPDATEs and potential route flaps, thereby affecting overall network stability and performance.</t>
 
-	  <t>To mitigate these effects, network operators should carefully manage the advertisement intervals of these dynamic metrics, ensuring they are set to avoid unnecessary churn. The default minimum interval for metrics change advertisement, set at 30 seconds, is designed to balance responsiveness with stability. However, in scenarios with higher sensitivity to route stability, operators may consider increasing this interval further to reduce the frequency of updates.</t>
+	  <t>To mitigate these effects, network operators should carefully manage the advertisement intervals of these dynamic metrics, ensuring they are set to avoid unnecessary churn. The default minimum interval for metrics change advertisement, set at 30 seconds, is designed to balance responsiveness with stability. However, in scenarios with higher sensitivity to route stability, operators may consider increasing this interval further to reduce the frequency of UPDATEs.</t>
 
 	  <t> Furthermore, operators should implement robust route damping and filtering policies to control the propagation of changes and minimize the impact on dependent routes. By acknowledging and planning for these broader impacts, the mechanism can be deployed more effectively, ensuring optimal performance without compromising network stability.</t>
+
+	  <!-- XXX JMH - You don't really mean "damping" here.  Route flap
+	  damping responds to high degrees of churn by suppressing the prefix
+	  (withdraw).  What's your intent here? -->
```

Comment:
will remove this sentence. 
originally it was intended to describe the policies when routes is withdrawn.

Reply:
Removed the damping context.  See next push.